RecoVertex/V0Producer/src/V0Fitter.h

```diff
@@ -47,18 +41,28 @@
 #include "Geometry/CommonDetUnit/interface/GeomDet.h"
 #include "Geometry/TrackerGeometryBuilder/interface/GluedGeomDet.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
-
 #include "DataFormats/TrackingRecHit/interface/TrackingRecHit.h"
 #include "DataFormats/BeamSpot/interface/BeamSpot.h"
-
-#include <string>
-#include <fstream>
+#include "CommonTools/CandUtils/interface/AddFourMomenta.h"
```

Comment:
it's better not to rely on nested non-obvious includes and add an include (for a MessageLogger) in the .cc in this case where the corresponding methods (LogError etc) are used.